Plugin/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 project(Plugin VERSION 1.0)
 
 set(HEADER_FILES
-    ${CMAKE_CURRENT_SOURCE_DIR}/src/SofaPython3/config.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/SofaPython3/config.h.in
```

Comment:
config.h.in will be automatically handled by the sofa_create_component_in_package_with_targets macro at the end of this file